File: Seccion 11 - Funciones en C++/Seccion_11_Python/11_ejercicio_09.py

horas = 0
minutos = 0
segundos = 0

def tiempo(total_seg):
    global horas, minutos, segundos
    
    horas = int(total_seg / 3600)
    total_seg %= 3600
    minutos = int(total_seg / 60)
    segundos = total_seg % 60

total_seg = int(input('Digite el número de segundos totales: '))

tiempo(total_seg)

print('El tiempo equivalente a la cantidad {} de segundos es: {:02d}:{:02d}:{:02d}'.format(total_seg, horas, minutos, segundos))

File: Seccion 11 - Funciones en C++/Seccion_11_Dart/11_ejercicio_09.dart

import 'dart:io';

class Tiempo {
  int horas, min, seg;

  Tiempo();
}

void tiempo(int totalSeg, Tiempo tiempo_hms) {
  tiempo_hms.horas = (totalSeg / 3600).floor();
  totalSeg %= 3600;
  tiempo_hms.min = (totalSeg / 60).floor();
  tiempo_hms.seg = totalSeg % 60;
}

void main() {

  int totalSeg;

  stdout.writeln('Digite el número total de segundos: ');
  totalSeg = int.parse(stdin.readLineSync());

  Tiempo tiempo_hms = new Tiempo();

  tiempo(totalSeg, tiempo_hms);

  print('El tiempo equivalente a la cantidad $totalSeg de segundos es: ${tiempo_hms.horas}:${tiempo_hms.min}:${tiempo_hms.seg}');
  
}